.github/workflows/main.yml

```diff
@@ -33,25 +73,32 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-pip-
 
-    - name: Upgrade packaging tools
-      run: python -m pip install --upgrade pip setuptools virtualenv wheel
-
     - name: Install dependencies
-      run: python -m pip install --upgrade codecov tox
+      run: |
+        python -m pip install --upgrade pip setuptools virtualenv wheel
+        python -m pip install --upgrade codecov tox tox-gh-actions
 
-    - name: Run tox targets for ${{ matrix.python-version }}
+    - name: Run tox
+      if: ${{ matrix.tox-env == 'N/A' }}
       run: |
-        ENV_PREFIX=$(tr -C -d "0-9" <<< "${{ matrix.python-version }}")
-        TOXENV=$(tox --listenvs | grep "^py$ENV_PREFIX" | tr '\n' ',') tox
```

Comment:
This will be filled in by `tox-gh-actions` and with https://github.com/ymyzk/tox-gh-actions/pull/60 it could specify everything and the `tox-env` workaround here could be removed.